File: docs/source/examples/CRUD/Update.java
// Open a thread-safe transaction.
realm.executeTransaction(r -> {
    // Get a dog to update.
    var dog = realm.where(Dog.class).findFirst();

    // Update some properties on the instance.
    // These changes are saved to the realm.
    dog.setName("Wolfie");
    dog.setAge(dog.getAge() + 1);
});

File: docs/source/examples/CRUD/Update.kt
// Open a thread-safe transaction.
realm.executeTransaction { r ->
    // Get a dog to update.
    val dog = realm.where<Dog>().findFirst()

    // Update some properties on the instance.
    // These changes are saved to the realm.
    dog.name = "Wolfie"
    dog.age = dog.age + 1
}
